### PR TITLE
Start items in REQUESTED status; move to LISTED and notify stores every 24 hours

### PR DESCRIPTION
**Pull request by lucashu1**

### CODE REVIEW

**Comment by lucashu1** [comment]
New items start off as REQUESTED, without any notification flags set.

Every 24 hours:
1. Move items to LISTED
2. Set store/item notification flags
3. Run store-notification cron job

File: src/server/util/storeHelpers.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import sqlHelpers from '../util/sqlHelpers.js';
+import errorHandler from '../util/errorHandler.js';
+import sendgridHelpers from '../util/sendgridHelpers.js';
+
+async function sendStoreownerNotificationEmail() {
```
Comment (Raghav-Maheshwari) [inline src/server/util/storeHelpers.js]: sorry this is pedantic but can we change the name of this function to `sendNotificationEmailsToStores`? Just so its clearer that It's sending multiple emails to multiple stores potentially (not just one)

Comment (Raghav-Maheshwari) [reply src/server/util/storeHelpers.js]: or something similar

Comment (lucashu1) [reply src/server/util/storeHelpers.js]: Yep, will do

**Pull request merged**